code/modules/reqs/supplypacks.dm

```diff
@@ -782,12 +782,12 @@ ARMOR
 /datum/supply_packs/armor/scout_cloak
 	name = "Scout Cloak"
 	contains = list(/obj/item/storage/backpack/marine/satchel/scout_cloak/scout)
-	cost = 50
+	cost = 35
 
 /datum/supply_packs/armor/sniper_cloak
 	name = "Sniper Cloak"
 	contains = list(/obj/item/storage/backpack/marine/satchel/scout_cloak/sniper)
-	cost = 50
+	cost = 35
```

Comment:
```suggestion
	cost = 30
```
30 points makes more sense considering this thing is a straight downgrade to scout cloak. You can't use it while moving, and so its straight up worse in every way..